contribute/markdown_cheat_sheet.md

```diff
@@ -320,20 +320,20 @@ Text
 
 ## Add "Related pages" to a page 
 
-If you want pages from the specific sections Your tasks, Your domain and Tool assembly to be shown as Related pages, list their `page_id`. If you want to list multiple related pages, make sure to put them in a list like this: [page_id1, page_id2]. 
+If you want pages from specific sections to be shown as Related pages, list their `page_id`. If you want to list multiple related pages, make sure to put them in a list like this: [page_id1, page_id2]. 
 
-An overview of all RDMkit pages (belonging to the sections listed above) and their `page_id` can be found in the [Website overview page](website_overview).
+An overview of all Infectious Diseases Toolkit pages (belonging to the sections listed above) and their `page_id` can be found in the [Website overview page](/contribute/website_overview).
 
 
 ```yml
 related_pages: 
-   your_tasks: [page_id1, page_id2]
-   your_domain: [page_id1, page_id2]
-   tool_assembly: [page_id1, page_id2]
+   section: [page_id1, page_id2]
+   section: [page_id1, page_id2]
+   section: [page_id1, page_id2]
```

Comment:
```suggestion
   pathogen_characterisation: [page_id1, page_id2]
   social_and_economic_impact: [page_id1, page_id2]
   human_biomolecular_data: [page_id1, page_id2]
   human_clinical_and_health_data: [page_id1, page_id2]
   showcase: [page_id1, page_id2]
```

Trying to already have the section names in place even though the page_id are not set yet.